### ISSUE TITLE
[A11y_Bug]: The accessibility checker reports learn more page is not loading

### ISSUE THREAD

**Issue by SuriyaKumar2402** [IBMa/equal-access#1070]
### Project

a11y checker

### Browser

Chrome

### Operating System

Windows

### Automated testing tool and ruleset

Accessibility Checker latest deployment

### Description

The Accessibility checker reports **learn more iframe** is broken. The contents are not loading.

I have attached a screenshot of the issue.
<img width="570" alt="accessibility learn more " src="https://user-images.githubusercontent.com/100849613/189112137-9010b502-22f0-41b1-af53-d99f82259a3a.png">

The issue is in the first file link. the link keeps changing for each report generation.
the second file link is a properly working link.
![Screenshot (91)](https://user-images.githubusercontent.com/100849613/189112991-0e2a63fc-0e3a-4636-89e1-6e1d261b9cb0.png)

the issue is in the **this.helpMap** object in **getHelp()** function of **ace-node.js** file.
[CDN link of ace-node file](https://cdn.jsdelivr.net/npm/accessibility-checker-engine@3.1.35/ace-node.js)

I have been facing this issue in **windows** & **Ubuntu** OS.
please resolve this issue ASAP.

### Steps to reproduce

The following issue is reproduced while running the accessibility checker in **vs code**.
I have used a local HTML sample file for input.

**gulpFile.js**
`let gulp =  require('gulp');
let path = require('path');
let checker = require('accessibility-checker');

gulp.task('accessibility', function(done){
    checker.getCompliance('file://'+path.resolve('index.html'),'outputFile');
    done()
});`

**aceconfig.js**

`module.exports = {
    ruleArchive: "latest",
    policies: ["WCAG_2_1","IBM_Accessibility"],
    failLevels: ["violation", "potentialviolation"],
    reportLevels: [
        "violation",
        "potentialviolation",
        "recommendation",
        "potentialrecommendation",
        "manual",
        "pass",
    ],
    outputFormat: ["html","json"],
    label: [process.env.TRAVIS_BRANCH],
    outputFolder:"reports",
  };`

And run the accessibility checker in **nodejs** by creating launch **JSON** file.

**Comment by aliunwala** [comment]
From triage meeting:
We are still investigating this issue. 

We did notice that your GET request has a malformed endpoint. It is missing /doc/en-us/ as part of the URL that it should have hit.

**Comment by tombrunet** [comment]
Not sure if it's the same issue, but the jest boilerplate isn't generating the help URL at all. There's certainly something going on related to the help that we need to fix.
